### ISSUE TITLE
cfcall

### ISSUE THREAD

**Issue by FileMakerStuff** [fmdynamix/my_prems_repository#1]
test de remontée d'un bug

**Comment by FmDynamix** [comment]
ok et ?

**Comment by FileMakerStuff** [comment]
juste pour que tu vois que tu peux avoir dans "issue" plusieurs types de remontées

**Comment by FileMakerStuff** [comment]
oui, les issues remontent, c’est le test que je faisais, mais chez toi puisque c’est TON github

Le 16 oct. 2015 à 17:44, FmDynamix notifications@github.com a écrit :

> ok eh ?
> 
> —
> Reply to this email directly or view it on GitHub.

Ugo Di Luca
ugo@dlsystems.fr
06.12.80.26.40
